### PR TITLE
Add sky_tool logs command

### PR DESCRIPTION
**Pull request by iansf**

### CODE REVIEW

File: sky/packages/sky/lib/sky_tool [hunk 1]
```diff
@@ -447,6 +488,27 @@ class AndroidDevice(object):
...
         subprocess.check_output(cmd)
 
+    def logs(self):
+        def do_logs():
+            cmd = [
+                self.adb_path,
+                'logcat',
+                '-s',
+                'sky',
+                'chromium',
```
Comment (abarth) [inline sky/packages/sky/lib/sky_tool]: This doesn't need to be:

'sky,chromium'

?

**Comment by abarth** [comment]
LGTM

File: sky/packages/sky/lib/sky_tool (hunk 1 above)
Comment (iansf) [inline sky/packages/sky/lib/sky_tool]: Let me check -- I pulled this invocation out of the docs, but it could be wrong.

Comment (iansf) [inline sky/packages/sky/lib/sky_tool]: The two invocations seem to give identical output, correctly including chromium lines.

It looks like -s is for silencing the default filter, and then everything after that is interpreted as the set of filters, which explains why both syntaxes work.

Comment (abarth) [inline sky/packages/sky/lib/sky_tool]: Neat.

**Pull request merged**